### PR TITLE
Added NoiseLevelTest and BlurLevelTest with testIsLowerOrEqual.

### PR DESCRIPTION
**Pull request by michalnicinski**

### CODE REVIEW

File: tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?php
+/**
```
Comment (Kryniol) [inline tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php]: remove please

File: tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    private $referenceBlurLevel;
+
+    /**
+     * @var BlurLevel
+     */
+    private $testedBlurLevel;
+
+    public function testIsLowerOrEqual()
+    {
+        $this->referenceBlurLevel = new BlurLevel(BlurLevel::LOW);
```
Comment (Kryniol) [inline tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php]: with those enums you can use simply
```php
$someBlur = BlurLevel::LOW();
```

File: tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+use PHPUnit\Framework\TestCase;
+use XSolve\FaceValidatorBundle\Result\BlurLevel;
+
+class BlurLevelTest extends TestCase
+{
+
+    /**
+     * @var BlurLevel
+     */
+    private $referenceBlurLevel;
```
Comment (Kryniol) [inline tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php]: you don't need those properties

File: tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        $this->testedBlurLevel = new BlurLevel(BlurLevel::MEDIUM);
+        $this->assertNotTrue($this->testedBlurLevel->isLowerOrEqual($this->referenceBlurLevel));
+
+        $this->testedBlurLevel = new BlurLevel(BlurLevel::HIGH);
+        $this->assertNotTrue($this->testedBlurLevel->isLowerOrEqual($this->referenceBlurLevel));
+
+
+        $this->referenceBlurLevel = new BlurLevel(BlurLevel::MEDIUM);
+
+        $this->testedBlurLevel = new BlurLevel(BlurLevel::LOW);
```
Comment (Kryniol) [inline tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php]: I'd group them and split to multiple test cases to improve the readability

File: tests/XSolve/FaceValidatorBundle/Result/NoiseLevelTest.php [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+ * Date: 9/15/17
+ * Time: 1:39 PM
+ */
+
+namespace tests\XSolve\FaceValidatorBundle\Result;
+
+use PHPUnit\Framework\TestCase;
+use XSolve\FaceValidatorBundle\Result\NoiseLevel;
+
+class NoiseLevelTest extends TestCase
```
Comment (Kryniol) [inline tests/XSolve/FaceValidatorBundle/Result/NoiseLevelTest.php]: same remarks

File: tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php [hunk 6]
```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: michal
+ * Date: 9/15/17
+ * Time: 1:39 PM
+ */
+
+namespace tests\XSolve\FaceValidatorBundle\Result;
```
Comment (krzysztof-ciszewski) [inline tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php]: This should be uppercase, the `tests` directory is aliased to `Tests\\` namespace in composer.jsn

File: tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php (hunk 1 above)
Comment (michalnicinski) [reply tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php]: done

File: tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php (hunk 6 above)
Comment (michalnicinski) [reply tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php]: done

File: tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php (hunk 3 above)
Comment (michalnicinski) [reply tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php]: done

File: tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php (hunk 2 above)
Comment (michalnicinski) [reply tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php]: done

File: tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php (hunk 4 above)
Comment (michalnicinski) [reply tests/XSolve/FaceValidatorBundle/Result/BlurLevelTest.php]: done

File: tests/XSolve/FaceValidatorBundle/Result/NoiseLevelTest.php (hunk 5 above)
Comment (michalnicinski) [reply tests/XSolve/FaceValidatorBundle/Result/NoiseLevelTest.php]: done

**Pull request merged**